pages/docs/modules/supported-services.adoc

```diff
@@ -0,0 +1,56 @@
+:_module-type: REFERENCE
+
+[id='supported-services_{context}']
+= Supported services
+
+[role='_abstract']
+{productname-long} supports the following services:
+
+[id="table-supported-services_{context}"]
+
+.Supported services
```

Comment:
I'll have to check and see which services are available upstream.  Right now, I think only `Jupyter` and `OpenVino` would stay on this list since everything else requires a RH Marketplace to install